### ISSUE TITLE
no credentials server URL error on Mac

### ISSUE THREAD

**Issue by musketyr** [testcontainers/testcontainers-java#851]
if I am running tests on Mac with Docker for Mac and I have _Securely store Docker logins in macOS keychain_ option ticked I get following error

```
org.testcontainers.shaded.org.zeroturnaround.exec.InvalidExitValueException: Unexpected exit value: 1, allowed exit values: [0], executed command [docker-credential-osxkeychain, get], output was 26 bytes:
no credentials server URL
```

When I untick the option the error is gone (and I can't even reproduce it again).

Version: Docker Community Edition: 18.06.0-ce-mac70 (26399)

See https://github.com/musketyr/grails-aws-plugin-with-localstack for sample app

**Comment by bsideup** [comment]
Hi @musketyr! Does it happen with Testcontainers 1.9.0-rc1 (available in JCenter)?

**Comment by musketyr** [comment]
I'm sorry but I can't reproduce it. Once I've unticked the option, even ticking it back and restarting won't cause the error again.

**Comment by toshiakiezaki** [comment]
I'm facing the same problem, even when I unflag and flag the _Securely store Docker logins in macOS keychain_ option. Tomorrow I'll update to 1.9.0-rc1 as @bsideup suggested and test again. 

If I get the same results, I'll try to collect more information about this issue and post them here. I suspect that the host information (required to get the credentials from _docker-credential-osxkeychain_) is not being passed to the process executor.

**Comment by toshiakiezaki** [comment]
Nevermind, after removing all config files and fresh installing Docker (enabling _Securely store Docker logins in macOS keychain_ and logging in with my account), I was able to run successfully my integration tests (with Testcontainers 1.8.3 and 1.9.0-rc1).

**Comment by kiview** [comment]
Okay, probably some weird effect with MacOS.
I'll close the issue for now.

**Comment by planetf1** [comment]
In case it helps, I noticed the same error

I was working not just with docker locally on the mac, and kubernetes, but also a cloud service. Since I didn't need local k8s for a while I stopped it. As I was working I realised I needed to rebuild a docker image on my mac. After doing this I did a 'docker login' and got the error above.

After reading here I decided to restart docker locally
After it had started I was then able to use docker login ok

Hope that helps
